R/stop-level_excel.R

```diff
@@ -191,7 +146,7 @@ hist_xls_run_checks <- function(df) {
   )
 
   # check for missing sci_name values, except for "Accipiter species"
-  hist_xls_sci_name_check(df)
+  # hist_xls_sci_name_check(df)
```

Comment:
Shall we exclude `accipiter species` from this data import altogether? Related to #106.

Reply:
Hm! Yeah, like 106 I think this can be excluded here. Avoids a later data-cleaning step

Reply:
👍🏼